prediction_market_agent/agents/microchain_agent/agents_nft_game.py

```diff
@@ -0,0 +1,76 @@
+from prediction_market_agent_tooling.gtypes import ChecksumAddress
+from pydantic import BaseModel
+from web3 import Web3
+
+from prediction_market_agent.agents.identifiers import AgentIdentifier
+
+
+class NFTAgent(BaseModel):
+    name: str
+    identifier: AgentIdentifier
+    wallet_address: ChecksumAddress
+    mech_address: ChecksumAddress
+
+    @property
+    def url(self) -> str:
+        return self.name.lower().replace(" ", "-")
+
+
+AGENTS_IN_THE_GAME = [
```

Comment:
I know it doesn't look like the best thing to have everything hard-coded here, but that's what PMA repository currently expects -- that each agent is defined in the code.